website/pages/docs/job-specification/expose.mdx

```diff
@@ -26,46 +26,100 @@ Service [check](https://nomadproject.io/docs/job-specification/service/#check-pa
 configurations can use their [expose](/docs/job-specification/service#expose)
 parameter to automatically generate expose path configurations for HTTP and gRPC checks.
 
+```hcl
+job "expose-check-example" {
+  datacenters = ["dc1"]
+
+  group "api" {
+    network {
+      mode = "bridge"
+    }
+
+    service {
+      name = "count-api"
+      port = "9001"
+
+      connect {
+        sidecar_service {}
+      }
+
+      check {
+        expose   = true
+        name     = "api-health"
+        type     = "http"
+        path     = "/health"
+        interval = "10s"
+        timeout  = "3s"
+      }
+    }
+
+    task "web" {
+      driver = "docker"
+
+      config {
+        image = "hashicorpnomad/counter-api:v2"
+      }
+    }
+  }
+}
+```
+
+For uses other than Consul service checks, use the `expose` configuration in the
+`proxy` stanza. The example below effectively demonstrates exposing the `/health`
+endpoint similar to the example above, but using the fully flexible `expose`
+configuration.
+
 ```hcl
 job "expose-example" {
   datacenters = ["dc1"]
-  group "dashboard" {
+
+  group "api" {
     network {
       mode = "bridge"
-      port "expose" {
+
+      port "api_expose_healthcheck" {
```

Comment:
Are we planning to publish these docs prior to 0.11.2? Would be nice to drop this since #7800 has been merged.

Reply:
I was kind of trying to make this example serve double-duty
1) show what the generated plumbing for service checks look like
2) show how a user would manually reference the port for an exposed path

The usefulness of 2 is dubious, certainly in the context of countdash. I'll update this to also use auto dynamic ports.